frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -680,15 +680,7 @@ class Worksheet extends React.Component {
             }
             shouldScroll = !isOnScreen(element);
         }
-        if (item && (!item.props || !item.props.item)) {
```

Comment:
Is this piece of code no longer needed?

Reply:
You can easily find there are no refs of children mode for worksheetItemList component, so the variable 'item' here is always undefined, which causes the line never be executed.